TeachingRecordSystem/src/TeachingRecordSystem.Core/Events/PersonsMergedEvent.cs

```diff
@@ -0,0 +1,26 @@
+namespace TeachingRecordSystem.Core.Events;
+
+public record PersonsMergedEvent : EventBase, IEventWithPersonAttributes
```

Comment:
We'll need to tweak this so that we can get it to appear in the Change History for both persons but can figure that one out later..